packages/discovery-react-components/src/components/SearchFacets/utils/searchFacetInterfaces.ts

```diff
@@ -92,3 +92,10 @@ export interface FieldFacetsByCategoryEntity {
     facets: SelectableFieldFacetWithCategory[];
   };
 }
+
+export enum FetchAggregationStates {
```

Comment:
I was thinking it made sense to have this definition here and then just import this type into `DiscoverySearch` since this is where the types related to the facets/aggregations are defined.